ee/ucp/admin/configure/restrict-services-to-worker-nodes.md

```diff
@@ -81,29 +71,38 @@ $ kubectl get <object> <workload> -o json | jq -r '.spec.template.spec.toleratio
 }
 ```
 
-### Allow Users and Service Accounts to Schedule on Manager / DTR Nodes
+### Allow users and service accounts to schedule workloads on manager / DTR nodes
 
-To allow Kubernetes Users and Service Accounts to deploy workloads accross all
-node types in your cluster, an Administrator will need to tick "Allow all
-authenticated users, including service accounts, to schedule on all nodes,
-including UCP managers and DTR nodes." in the UCP web interface. 
+To allow Kubernetes users and service accounts to deploy workloads across all node types in a cluster, an administrator must select the **Allow all authenticated users, including service accounts, to schedule on all nodes, including UCP managers and DTR nodes.** check box in the UCP UI. 
 
 ![](../../images/restrict-services-to-worker-nodes-3.png){: .with-border}
 
-For all new workloads deployed by Kubernetes Users after this box has been
-ticked, UCP will apply a toleration to your workloads to allow the pods to be
-scheduled on all node types. For existing workloads, the User would need to edit
-Pod Specification as detailed above in the "Allow Administrators to Schedule on
-Manager / DTR Nodes" section.
+When this option is enabled, for all new workloads deployed by Kubernetes users, UCP will apply a toleration to workloads to allow the pods to be scheduled on all node types. 
+
+For existing workloads, users will need to edit the pod specification, using `kubectl edit <object> <workload>` or the UCP UI, and add the following toleration:
+
+```bash
+tolerations:
+- key: "com.docker.ucp.manager"
+  operator: "Exists"
+```
+
+To verify that the toleration was applied successfully:
+
+```bash
+$ kubectl get <object> <workload> -o json | jq -r '.spec.template.spec.tolerations | .[]'
+{
+  "key": "com.docker.ucp.manager",
+  "operator": "Exists"
+}
+```
 
-There is a NoSchedule taint on UCP managers and DTR nodes and if you have
-scheduling on managers/workers disabled in the UCP scheduling options, then a
-toleration for that taint will not get applied to the deployments, so they
-should not schedule on those nodes. Unless the Kube workload is deployed in the
-`kube-system` name space.
+> **Note**
+>
+> There is a `NoSchedule` taint value available on UCP managers and DTR nodes. If the option to schedule managers and DTR nodes is disabled, then a toleration for that taint will not get applied to the deployments. Unless the Kubernetes workload is deployed in the `kube-system` namespace, the workloads are not scheduled on those nodes.
```

Comment:
Suggest removing "then" (before "a toleration") and changing "get applied" to "be applied".

Also, I suggest editing the last sentence to read: "Note that workloads are only scheduled on the nodes if the Kubernetes workload is deployed in the `kupe-system` namespace."